src/lib/Search.svelte

```diff
@@ -168,7 +177,8 @@
           <img src={preview.image} class="dropdown-item-image" alt="Map Thumbnail" />
           <div class="dropdown-item-text">
             {preview.name}<br />
-            <div class="dropdown-item-text2">{preview.uploader}</div>
```

Comment:
You inherited these class names and continued the pattern, but I think using something like:

- `dropdown-item-map-name`
- `dropdown-item-uploader`
- `dropdown-item-stats`